ibm/service/power/resource_ibm_pi_network.go

```diff
@@ -199,7 +199,17 @@ func resourceIBMPINetworkCreate(ctx context.Context, d *schema.ResourceData, met
 		body.Gateway = gateway
 		body.Cidr = networkcidr
 	}
-
+	wsclient := instance.NewIBMPIWorkspacesClient(ctx, sess, cloudInstanceID)
+	wsData, err := wsclient.Get(cloudInstanceID)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	if wsData.Capabilities[PER] {
```

Comment:
What if Capabilities `PER` does not exist? Will it always be a boolean or could be empty?